test/cpp/ios/run_tests.sh

```diff
@@ -20,6 +20,11 @@ set -ev
 
 cd "$(dirname "$0")"
 
+echo "TIME:  $(date)"
+
+./build_tests.sh
```

Comment:
You're running ``./build_tests.sh` here, but it it looks like build_tests.sh was renamed to build_and_run_tests.sh: 
test/cpp/ios/build_tests.sh → test/cpp/ios/build_and_run_tests.sh

Reply:
Ouch. Thanks for catching this. I renamed the wrong file :(. It should be `run_tests.sh` → `build_and_run_tests.sh`